README.md

```diff
@@ -150,7 +150,7 @@ acquire this dependency.
 6. Run `.\vcpkg.exe integrate install` which tells Visual Studio which vcpkg instance you wish to use. If you have never
    done this before, you may be prompted to elevate.
 7. Open Visual Studio, and choose the "Clone or check out code" option. Enter the URL to this
-   repository, typically `https://github.com/microsoft/STL`
+   repository, typically `https://github.com/microsoft/STL`. Make sure that submodules are also checked out.

```

Comment:
Is there a checkbox or something for this? If not we need an extra step that explains how to do that.